pkg/operator/periodicbackupcontroller/periodicbackupcontroller.go

```diff
@@ -80,6 +89,16 @@ func (c *PeriodicBackupController) sync(ctx context.Context, _ factory.SyncConte
 		return fmt.Errorf("PeriodicBackupController could not list backup CRDs, error was: %w", err)
 	}
 
+	if len(backups.Items) == 0 && !c.isDefaultBackup {
```

Comment:
can we go back to how we initially have this set in the loop?

Reply:
so the reason I have used this approach, is to avoid having to install a CR for the NoConfig 

Since this feature introduces NoConfig backups, the user should not be required to apply any CR 

please correct me if I am wrong